src/patternfly/components/Dropdown/dropdown-basic.hbs

```diff
@@ -1,3 +1,2 @@
 <div class="pf-c-dropdown__menu" {{#unless dropdown--IsExpanded}}hidden{{/unless}}>
```

Comment:
should we add `--modifier` and `--attribute` and `{{> @partial-block}}` to this? Also it's worth noting that this menu is empty in the example code. Not sure if fixing that is in this scope of this PR or not.

Reply:
I'd say yes for consistency sake.

Reply:
added modifier, attribute and partial-block. Looks like this file if for the dropdown-basic element, but it uses pf-c-dropdown__menu  as the class, which already has its own file. So should I remove this one?